http/auth_service.go

```diff
@@ -85,6 +86,8 @@ type authResponse struct {
 	User        string               `json:"user"`
 	Permissions []permissionResponse `json:"permissions"`
 	Links       map[string]string    `json:"links"`
+	CreatedAt   time.Time            `json:"createdAt"`
```

Comment:
you can use platform. CRUDLog  here

Reply:
We ended up using `CreatedAt` and `UpdatedAt` here to be more explicit in the methods that we're using. To me, `CRUD` connotes that the operation has something to do with the Creation, Reading, Updating, and Deletion of data. Since we only care about two of those conditions (`create`, `update`), it felt important to explicit in the intent of the struct

Reply:
We ended up using CreatedAt and UpdatedAt here to be more explicit in the methods that we're using. To me, CRUD connotes that the operation has something to do with the Creation, Reading, Updating, and Deletion of data. Since we only care about two of those conditions (create, update), it felt important to explicit in the intent of the struct